prompt2model/utils/parse_responses.py

```diff
@@ -13,10 +14,12 @@
 
 
 def find_and_parse_json(
-    response: openai.Completion, required_keys: list, optional_keys: list
+    response: openai.Completion, required_keys: list, optional_keys: list = []
 ) -> dict | None:
     """Parse stuctured fields from the API response.
 
+    Incase there are multiple json objects in the response, we take the rightmost one
```

Comment:
"Incase" should be "in case". And add a period at the end.

```suggestion
    In case there are multiple JSON objects in the response, take the final one.
```